ChangeLog

```diff
@@ -4,14 +4,19 @@ Pydot versions since 1.4.2 adhere to [PEP 440-style semantic versioning]
 (https://www.python.org/dev/peps/pep-0440/#semantic-versioning).
 
 
-2.0.1 (unreleased)
+3.0.0 (unreleased)
 ------------------
 
 Added:
+- Breaking change: support for Python 3.7 dropped totally. (#371)
+  It was already dropped, but now it's official.
+- Breaking change (but for nobody): Old unknown syntax removed (#377).
+  A "port angle" that takes the form `@<id>`
+  and an alternative form of a port `:(<id>,<id>)` deleted.
```

Comment:
```suggestion
- Breaking change (but for nobody): Invalid syntax removed (#377).
  Removed syntax rules that were never implemented in graphviz's own parser.
```

(IMHO the specifics of the removed syntax are unnecessary detail. Nobody could use it before, and they can't use it now. But, your call.)